includes/class-wc-stripe-order-helper.php

```
@@ -484,6 +505,81 @@ public function update_stripe_setup_intent_id( ?WC_Order $order = null, string $
+	/**
+	 * Updates the Stripe presentment amount for order.
+	 *
+	 * @since 10.5.0
+	 *
+	 * @param WC_Order|null $order
+	 * @param string $presentment_amount
+	 * @return false|void
+	 */
+	public function update_stripe_presentment_amount( ?WC_Order $order = null, string $presentment_amount = '' ) {
+		return $this->update_order_meta( $order, self::META_STRIPE_PRESENTMENT_AMOUNT, $presentment_amount );
+	}
```

PHPMD (2.15.0)

[error] The method update_stripe_presentment_amount is not named in camelCase. (undefined)

(CamelCaseMethodName)